### DISCUSSION TITLE
Botallen repository

### DISCUSSION THREAD

**Discussion by KolDe001** [community/community#43093] [Repositories]
### Select Topic Area

Product Feedback

### Body

It's not working properly. frequently stucked and unable to resolve the channels issue.
Please check and assist to resolve the issues in Kodi.botallen.com  in version 19.3 and 19.4 and 19.5 also in Alpha 1, Alpha 2, Alpha 3 for Android TV version 9.0.

**Comment by airtower-luna** [comment]
It looks like you're looking for help with a software hosted on GitHub, maybe https://github.com/botallen/repository.botallen? Normally in that case you should open an issue in the repository itself (instead of the GitHub community discussions). Unfortunately I see that that particular repository is marked as no longer maintained (the owner has declared they won't work on it any more), so there isn't any place really. If you can't fix the problems yourself, the best option is probably to look for alternatives.

**Comment by KolDe001** [comment]
Dear Team,

Please try to find out solution regarding BotAllen Repository

We are habitual with this application and it's not working so we are
mentally disturbed.

Regards,
Indian Citizen
